x/staking/spec/03_messages.md

```diff
@@ -34,7 +34,7 @@ in the first end-block.
 ## Msg/EditValidator
 
 The `Description`, `CommissionRate` of a validator can be updated using the
-`Msg/EditCandidacy` service message.
+`Msg/EditValidator` service message.
```

Comment:
I think we should avoid having the slash after `Msg` after #9139, to avoid confusion.

```suggestion
`MsgEditValidator` message.
```